docs/DeveloperGuide.md

```diff
@@ -262,27 +262,36 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+Researchers who..
+* have a need to manage a significant number of study participants
+* need an easy way to contact different experimental groups
```

Comment:
This doesn't seem to be covered by our intended feature(s). Do you think we should remove it from the user profile, or at least modify it a little to better fit the vision for the project?

Reply:
hmmm i feel like it is covered by our features tbh. cos u can filter list of contacts by study group then export that list of contacts (i think?)

Reply:
Yup I think you might be right. While our feature doesn't cover the entire point, it does facilitate part of it. Let's keep it for now. 

Thank you @PrishaVP